### PR TITLE
chore(internet_port): added new ports and removed unnecessary string class

### PR DESCRIPTION
**Pull request by anirudh-hegde**
### SUMMARY
Added new ports and removed unnecessary string class

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=6

### CODE REVIEW

**Comment by john-bodley** [comment]
Thanks @anirudh-hegde for the change. @cccs-tom would you mind taking a look at this?

**Review by cccs-tom — APPROVED** [review]
Lgtm!

**Review by villebro — APPROVED** [review]
LGTM

**Comment by villebro** [comment]
@anirudh-hegde please check the linting errors:
image

**Comment by villebro** [comment]
@anirudh-hegde there's still that trailing whitespace issue:
```
superset/advanced_data_type/plugins/internet_port.py:93:0: C0303: Trailing whitespace (trailing-whitespace)
```

**Comment by anirudh-hegde** [comment]
image
It means in line 91 - 93 there is a trailing whitespace?

File: superset/advanced_data_type/plugins/internet_port.py [hunk 1]
```diff
@@ -83,9 +88,9 @@ def port_translation_func(req: AdvancedDataTypeRequest) -> AdvancedDataTypeRespo
                 else port_conversion_dict[string_value]
             )
         except (KeyError, ValueError):
-            resp["error_message"] = str(
+            resp["error_message"] = \
                 f"'{string_value}' does not appear to be a port name or number"
-            )
+
```
Comment (villebro) [inline superset/advanced_data_type/plugins/internet_port.py:93]: This PR adds a newline here which the linter wants to be removed
```suggestion
```

**Comment by anirudh-hegde** [comment]
I used `pylint internet_port.py`
image

**Comment by anirudh-hegde** [comment]
Ohh my bad!! I merged branch 'apache:master' into spear

File: superset/advanced_data_type/plugins/internet_port.py [hunk 2]
```diff
@@ -48,6 +48,11 @@
     "ldaps": [636],
     "imap2": [143],  # aka imap
     "imaps": [993],
+    "MySQL": [3306],
+    "Remote Desktop": [3389],
+    "dns": [53],
+    "ctf": [84],
+    "pdap": [344]
```
Comment (villebro) [inline superset/advanced_data_type/plugins/internet_port.py:55]: This is still failing:
```suggestion
    "pdap": [344],
```

**Comment by anirudh-hegde** [comment]
Should I need to change the code?

**Comment by rusackas** [comment]
Looks like you just need to run the [pre-commit hook](https://superset.apache.org/docs/contributing/hooks-and-linting/) and you should be OK.

**Comment by anirudh-hegde** [comment]
> Looks like you just need to run the [pre-commit hook](https://superset.apache.org/docs/contributing/hooks-and-linting/) and you should be OK.

Hey @rusackas,
image
I used pre-commit run --all-files

**Comment by anirudh-hegde** [comment]
> Looks like you just need to run the [pre-commit hook](https://superset.apache.org/docs/contributing/hooks-and-linting/) and you should be OK.

You were right.
Thank you, @rusackas

**Pull request merged**